README.md

```diff
@@ -90,6 +104,9 @@ The following environment variables represent the configuration of the actual Sh
 | `GSUITE_TOKEN` | | GSuite authentication token | `gsuite` |
 | `SHERIFF_GSUITE_DOMAIN` | | The primary domain of your GSuite account | `gsuite` `slack` |
 | `SHERIFF_SLACK_DOMAIN` | | The "domain" part of `{domain}.slack.com` for your Slack instance  | `gsuite` if you add slack email addresses to your google groups for notifications |
+| `ORGANIZATION_NAME` | | GitHub org that the GitHub App has been installed to | `electron` |
+| `REPO_NAME` | | GitHub repo that the GitHub App has been installed to  | `electron` |
```

Comment:
Wondering if these variables should ever be different from `PERMISSIONS_FILE_ORG` and `PERMISSIONS_FILE_REPO` 🤔 

i.e. I don't think there's a usecase for the github app to be installed in a different org than the one containing the `permissions` repo.

Would your setup have these vars as the same thing? If so we should probably just dedupe them 😅